spaceapi_server/src/lib.rs

```diff
@@ -15,9 +16,10 @@ use std::sync::Mutex;
 use std::sync::Arc;
 
 use rustc_serialize::json::{Json, ToJson};
-use iron::{Request, Response, IronResult, Iron, Set};
+use iron::prelude::*;
```

Comment:
Yeah, but it confuses e.g. static code analysis tools :) "Namespaces are one honking great idea."